python/ray/data/read_api.py

```diff
@@ -628,6 +630,9 @@ def read_images(
         filesystem: The filesystem implementation to read from.
         parallelism: The requested parallelism of the read. Parallelism may be
             limited by the number of files of the dataset.
+        ray_remote_args: kwargs passed to ray.remote in the read tasks.
+        arrow_open_file_args: kwargs passed to
+            ``pyarrow.fs.FileSystem.open_input_file``.
```

Comment:
nit: can we follow the same ordering, to add these two parameters after `meta_provider`?

Reply:
Ah yep, looks like these got reordered when resolving merge conflicts, I'll fix that!